configuration/cdap-env.xml

```diff
@@ -24,11 +50,8 @@
     <value>
 export JAVA_HOME={{java64_home}}
 if [ "$JAVA_HOME" != "" ]; then
-  #echo "run java in $JAVA_HOME"
   JAVA_HOME=$JAVA_HOME
-fi
-
-if [ "$JAVA_HOME" = "" ]; then
+else
```

Comment:
Fair enough. This whole junk pile was copied from the Ambari HBase service. I'll clean it up in the next PR for config.